CHANGELOG.md

```diff
@@ -1,5 +1,28 @@
+## 0.2.0 (2022-11-16)
+
+
+### Features
+
+* add basic markdown support ([#84](https://github.com/awslabs/duvet/issues/84)) ([f8ebf29](https://github.com/awslabs/duvet/commit/f8ebf298c6dca3c2a261d6a3fbc3703dd1c6703b))
+
+
+### Bug Fixes
+
+* add word boundary assertions for extracted keywords ([#72](https://github.com/awslabs/duvet/issues/72)) ([02c9245](https://github.com/awslabs/duvet/commit/02c92452158debf1be82c702824689ab01b08aa0))
```

Comment:
Are you sure this was generated right? The bug fixes are showing up for both 0.2 and 0.1

Reply:
Like I mean all these commits were already listed under 0.1 as bug fixes